addons/survey/tests/test_certification_flow.py

```diff
@@ -115,6 +115,16 @@ def test_flow_certification(self):
         self.assertEqual(user_inputs.scoring_percentage, 87.5)
         self.assertTrue(user_inputs.scoring_success)
 
+        # assert statistics
+        statistics = user_inputs._prepare_statistics()[user_inputs]
+        total_statistics = statistics['totals']
+        self.assertEqual(total_statistics, [
```

Comment:
Shouldn't we use 'sorted' to be sure we compare list content and not their order ? It will probably use the field order, but better safe than sorry with that kind of check.

Reply:
It iterates on o2m (looked quickly at method) to prepare data, so I suppose the final ordering will be based on model order. Looked reaaaaally quick so maybe not right, but anyway sorting is always better.